codec-http2/src/test/java/io/netty/handler/codec/http2/Http2FrameCodecTest.java

```diff
@@ -222,10 +223,22 @@ public void flowControlShouldBeResilientToMissingStreams() throws Http2Exception
         Http2FrameCodec codec = new Http2FrameCodec(enc, dec, new Http2Settings(), false);
         EmbeddedChannel em = new EmbeddedChannel(codec);
 
-        // We call #consumeBytes on a stream id which has not been seen yet to emulate the case
-        // where a stream is deregistered which in reality can happen in response to a RST.
-        assertFalse(codec.consumeBytes(1, 1));
+        AtomicReference<Http2Exception> errorRef = new AtomicReference<>();
+        em.eventLoop().execute(() -> {
```

Comment:
we need to call this on the `EventLoop` as the codec asserts that we are on the event loop.